langchain4j/src/main/java/dev/langchain4j/service/DefaultAiServices.java

```diff
@@ -192,8 +194,7 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Exceptio
 
                         response = Response.from(response.content(), tokenUsageAccumulator, response.finishReason());
 
-                        Object parsedResponse;
-                        parsedResponse = serviceOutputParser.parse(response, returnType);
+                        Object parsedResponse = serviceOutputParser.parse(response, returnType);
```

Comment:
Minor fix, couldn't see the reason for splitting into two lines. Seems like an old refactor that didn't go all the way?